udaykalra/portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/** Creates an <li> element containing text. */
+function createCommentList(inputText) {
+  const listElement = document.createElement('li');
+  listElement.innerText = inputText;
+  return listElement;
+}
+
 /**
- * Adds a random greeting to the page.
+ * Fetches comments for display.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function getCommentData() {
+  fetch('/data')                          // sends a request to /data
+      .then(response => response.json())  // parses the response as JSON
+      .then((myComments) => {  // now we can reference the fields in myComments!
```

Comment:
`comments` seems sufficiently clear as the variable name (the "my" doesn't seem to contribute to clarity, since there is only one comments variable in this scope). "my" might be useful in a situation where there are multiple comments variables, e.g.

```
let myComments = "foo";
let theirComments = "bar";
...
```